Task 1/POSAND/Question.md

```diff
@@ -0,0 +1,28 @@
+A permutation p1,p2...pN of {1,2,...,N} is beautiful if pi&pi+1 is greater than 0 for every 1≤i<N . You are given an integer N, and your task is toconstruct a beautiful permutation of length N or determine that it's impossible.
+
+Note that a&b denotes the bitwise AND of a and b.
+
+Input:
```

Comment:
Here as well, if possible! It doesn't stop me from merging your PR, but I think its a small change which you would be able to do!